trick_sims/SIM_balldrop/README.md

```diff
@@ -0,0 +1,45 @@
+# New Years Eve Balldrop Simulation
+
+The New Years Eve Ball is 75 feet high and takes 60 seconds to descend. This simulation allows a demonstration of the force of gravity assuming all other forces are negligable.
+
+![Ball Drop - Initial Conditions](Images/BallDrop.png)
+
+## Building the Simulation
+
+1. Build the simulation excecutable.
+   
+```bash
+cd SIM_balldrop
+```
+
+```bash
+trick-CP
+```
+
+3. Build the graphics client.
+   
+```bash
+cd models/graphics
+```
+
+```bash
+make
+```
+
+## Running the Simulation
+
+1. Run the simulation .exe accoording the the input file.
+   
+```bash
+cd ../../../SIM_balldrop
+```
+
+```bash
+./S_main_*.exe RUN_test/input.py
+```
+
+## Dynamics Model
+
+The force acting on our ball will be gravity only.
+
+To determine the position of the ball, we use the fundamental kinematics equations that $v = at\$ and $\text{pos} = \frac{1}{2}at^2 \$
```

Comment:
Consider adding a description of each variable that plays into the physics of the simulation, the default for the variable, and a hint as to how to change the default if desired.